premailer/tests/test_premailer.py

```diff
@@ -2302,3 +2302,28 @@ def test_turnoff_cache_works_as_expected(self):
         result_html = p.transform()
 
         compare_html(expect_html, result_html)
+
+    def test_links_without_protocol(self):
+        """If you the base URL is set to https://example.com and your html
+        contains <a href="//otherdomain.com/">...</a> then the URL to point to
+        is "https://otherdomain.com/" not "https://example.com/file.css"
+        """
+        html = """<html>
+        <head>
+        </head>
+        <body>
+        <a href="//example.com">Link</a>
```

Comment:
Hmm, but we're not calling ```make_links_absolute ``` as far as I can tell, so I don't really get it...?

And sometimes (probably less and less frequently but it still happens), http is the right choice for performance...